app/src/main/java/com/metrics_board/app/service/roll/ProjectService.java

```diff
@@ -36,6 +40,30 @@ public ProjectResponse getProject(UUID ownerId, Long id) throws ResourceNotExist
         throw new ResourceNotExistException();
     }
 
+    public List<ProjectResponse> getProjects(UUID ownerId) {
+        List<Project> foundProjects = projectRepository.findAllByOwnerId(ownerId);
+        List<ProjectResponse> projectResponseList = new ArrayList<>();
+
+        for (Project project : foundProjects) {
+            projectResponseList.add(createProjectResponse(project));
+        }
+
+        return projectResponseList;
+    }
+
+    public void deleteProject(UUID ownerId, Long id) {
+        Optional<Project> foundProject = projectRepository.findById(id);
+        if (foundProject.isPresent()) {
+            if (foundProject.get().getOwnerId().equals(ownerId)) {
+                projectRepository.deleteById(id);
+            } else {
+                log.warn(ownerId + " tried to delete id " + id + " that is not owned by");

```

Comment:
Instead of concatenation better to use templates to build log message
`log.warn("{} tried to delete project id {} that is not owned by", ownerId,  id);`